labelbox/schema/slice.py

```diff
@@ -0,0 +1,64 @@
+from typing import List
+
+from labelbox.orm.db_object import DbObject
+from labelbox.orm.model import Field
+
+
+class Slice(DbObject):
+    """
+    A Slice is a saved set of filters (saved query) used to find data rows matching a certain criteria
```

Comment:
Let's add a comment noting that this is an abstract class, and users should not instantiate this class